docs/pages/guides/using-bun.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Using Bun
+description: A guide on using Bun with Expo.
+---
+
+[Bun](https://bun.sh/) is a JavaScript runtime and a drop-in alternative for [Node.js](https://nodejs.org/en). On Expo projects, Bun can be used to install npm packages and run Node scripts. The benefits of using Bun is that it is measurably faster at installing packages than yarn/pnpm/npm, and at least 4x faster than Node.js, which can give a huge boost to your local development experience.
+
+## Staring a new Expo project with Bun
+
+To create a new app using Bun, first [install Bun on your local machine](https://bun.sh/) with:
+
+```bash
+curl -fsSL https://bun.sh/install | bash
+```
+
+Now use it to create your new Expo project
+
+```bash
+bun create expo my-app
+```
+
+You can also run any **package.json** script with `bun run`
+
+```bash
+bun run ios
+```
+
+And install packages with `bun expo install`
+
+```bash
+bun expo install expo-av
+```
+
+## Using Bun for EAS builds
+
+EAS will decide which packager to use based the lockfile in your codebase. So if you want EAS to use Bun, run `bun install` in your codebase and ensure it creates a **bun.lockb** - the Bun lockfile. Make sure to delete any other lockfiles. As long as this lockfile is in your codebase, Bun will be used as the package manager for your builds.
+
+### Customizing your Bun version on EAS
+
+EAS will use `bun@1.0.2` by default. If you want or need to use a particular version of Bun, you can configure the exact version in each build in your **eas.json**.
+
+```json eas.json
+{
+  "build": {
+    "test": {
+      "bun": "1.0.0"
+      // other settings...
+    }
+    // other build profiles...
+  }
+}
+```
+
+## Migrate to using Bun from yarn, npm or pnpm
+
+It is currently not possible to import another package manager's lockfile into Bun (though this feature is bieng [worked on](https://github.com/oven-sh/bun/issues/1751)). Until this is done, there is an element of risk to switching over to Bun on an existing project: you will have to remove your existing lockfile and `bun install` based on your **package.json**.
+
+The purpose of a lockfile is to _lock_ down your dependency tree. If there are any packages in **package.json** whose version numbers start with a `^` or `~`, you are likely to end up with a different version of the package. `^` means you're opting into future minor and patch versions, `~` means you're opting into future patch versions only. A lot of the time, you will not encounter problems, as according to SemVer minor and patch version shoudn't include any breaking changes. Unfortuantely things can easily slip through and we should always expect breaking changes to be a possibility. Your lockfile locks fown the _specific_ version and you won't get any updates unless you explicitly opt in. By deleting your lockfile, you are losing that safety net and getting the latest versions available of all the packages as defined in your **package.json**.
```

Comment:
```suggestion
The purpose of a lockfile is to _lock_ down your dependency tree. If there is a library in **package.json** whose version number starts with a `^` or `~`, you are likely to end up with a different version of the package. 

- `^` means you're opting into future minor and patch versions
- `~` means you're opting into future patch versions only

According to Semantic Versioning (SemVer), minor and patch versions do not include breaking changes. Unfortunately, breaking changes can still slip through. Since a lockdown file contains **specific versions** of dependencies, you will not get updates unless you explicitly opt-in. By deleting the lockfile, you are losing that safety and getting the latest versions available of all the packages as defined in your **package.json**.
```

- The previous version of this reads a bit like a wall of text, as I think it contains important conceptual info, it was harder to skim through. Let's break it down.
- Fixed typos
- Removed nagation contraction (shouldn't)
- Also clarified what SemVer is by using its full form
- Fixed the switch between first and second-person voice as per our [writing styelguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person)
- I think the part where breaking changes and SemVer being mentioned was a bit verbose. Changed it to be a bit more concise.